php/siler-swoole/composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+      "leocavalcante/siler": "dev-master"
```

Comment:
could you use `~1.7.2` instead of `dev-master` ?

Reply:
@leocavalcante it is ok?

Reply:
@RicardoSette, no problem. The latest stable release supports features used in the test.

Reply:
@RicardoSette @leocavalcante version will be in-sync :heart: Thanks to `dependabot` and `renovate`

Reply:
@waghanza how cool! Just added to Siler itself, thanks for the hint.